int8 sudocubeNo
int16 obs1x
int16 obs1y
int16 obs2x
int16 obs2y
---